filters.go

```diff
@@ -169,6 +169,44 @@ func (p *Pipe) Join() *Pipe {
 	return Echo(output + terminator)
 }
 
+// Last reads from the pipe, and returns a new pipe containing only the last N
+// lines. If there is an error reading the pipe, the pipe's error status is also
+// set.
+func (p *Pipe) Last(lines int) *Pipe {
+	if p == nil || p.Error() != nil {
+		return p
+	}
+	
```

Comment:
You'll notice that there are no blank lines within any functions or methods in the `script` package. I tend to think Go code is more readable the more of it you can see at once! For consistency of style, would you mind removing the blank lines?